docker/staging/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.3'
+services:
+  coordinator:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    networks:
+      - labgrid_net
+    tty: true
+    entrypoint: "crossbar start"
```

Comment:
Why not use `command` instead, so it is possible to start image with alternative command (for debugging purposes)?